src/controllers/issuer.ts

```diff
@@ -3,12 +3,11 @@ import { check, param, validationResult } from 'express-validator';
 import { fromString } from 'uint8arrays';
 import type { DIDDocument, Service, VerificationMethod } from 'did-resolver';
```

Comment:
All imports here are available through @cheqd/sdk.

Let's avoid bringing unnecessary indirect dependencies to direct.
```suggestion
```